particles/Products/source/ProductItem.js

```diff
@@ -105,7 +105,9 @@ defineParticle(({DomParticle, resolver, html}) => {
         const resolvedImage = resolver ? resolver(product.image) : product.image;
         return Object.assign({
             resolvedImage,
-            styleBackground: `background-image:url(${resolvedImage})`
+            styleBackground: `background-image:url(${resolvedImage})`,
+            // TODO(sjmiles): subid required by renderer, not used in template (iirc, verify!)
```

Comment:
is this TODO still needed? the `subId` was added in many particles, so i think you might have already resolved the question.

Reply:
It's a new requirement. It's easy to get stuck wondering why some recipe doesn't render properly, so I left reminders here and there. Also, it's a requirement I'd like to remove asap.